mayastor/src/core/bdev.rs

```diff
@@ -171,9 +171,9 @@ impl Bdev {
                 }
             }
             spdk_sys::SPDK_BDEV_EVENT_RESIZE => {
-                info!("Received resize event for bdev {}", bdev.name())
+                warn!("Received resize event for bdev {}", bdev.name())
```

Comment:
According to [Tracing docs](https://tracing.rs/tracing/struct.level) `warn` is conventionally used only for hazardous situations. This seems to be a normal system oepration, so perhaps `info` was indeed appropriate?

Reply:
I forgot to note that in the commit message, I changed it to a warning because it's unimplemented. I'll add the author as a reviewer.